### PR TITLE
Add a more readable tooltip on source status

### PR DESCRIPTION
**Pull request by CFarcy**

### CODE REVIEW

File: src/modules/RA/DataSource/components/StatusChip/Tooltip.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Divider from '@material-ui/core/Divider';
```
Comment (mabhub) [inline src/modules/RA/DataSource/components/StatusChip/Tooltip.js:8]:
```suggestion
import {
  Divider,
  List,
  ListItem,
  ListItemIcon,
  ListItemText,
  makeStyles,
} from '@material-ui/core';
```

File: src/modules/RA/DataSource/components/StatusChip/Tooltip.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+import FlashOnOutlinedIcon from '@material-ui/icons/FlashOnOutlined';
+import TimerOutlinedIcon from '@material-ui/icons/TimerOutlined';
+import AddCircleOutlineOutlinedIcon from '@material-ui/icons/AddCircleOutlineOutlined';
+import DeleteOutlinedIcon from '@material-ui/icons/DeleteOutlined';
+import AssessmentOutlinedIcon from '@material-ui/icons/AssessmentOutlined';
+import CreateOutlinedIcon from '@material-ui/icons/CreateOutlined';
+import WarningOutlinedIcon from '@material-ui/icons/WarningOutlined';
+import PlayArrowOutlinedIcon from '@material-ui/icons/PlayArrowOutlined';
+import DoneAllOutlinedIcon from '@material-ui/icons/DoneAllOutlined';
```
Comment (mabhub) [inline src/modules/RA/DataSource/components/StatusChip/Tooltip.js:17]:
```suggestion
import {
  FlashOnOutlined as FlashOnOutlinedIcon,
  TimerOutlined as TimerOutlinedIcon,
  AddCircleOutlineOutlined as AddCircleOutlineOutlinedIcon,
  DeleteOutlined as DeleteOutlinedIcon,
  AssessmentOutlined as AssessmentOutlinedIcon,
  CreateOutlined as CreateOutlinedIcon,
  WarningOutlined as WarningOutlinedIcon,
  PlayArrowOutlined as PlayArrowOutlinedIcon,
  DoneAllOutlined as DoneAllOutlinedIcon,
} from '@material-ui/icons';
```

File: src/modules/RA/DataSource/components/StatusChip/Tooltip.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  translate,
+  status,
+  started,
+  ended,
+  errors,
+  total,
+  added_lines: added,
+  deleted_lines: deleted,
+  modified_lines: modified,
+}) {
```
Comment (mabhub) [inline src/modules/RA/DataSource/components/StatusChip/Tooltip.js:35]:
```suggestion
const TooltipStatus = ({
  translate,
  status,
  started,
  ended,
  errors,
  total,
  added_lines: added,
  deleted_lines: deleted,
  modified_lines: modified,
}) => {
```

File: src/modules/RA/DataSource/components/StatusChip/Tooltip.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            <ListItem className={classes.nested}>
+              <ListItemIcon><DeleteOutlinedIcon fontSize="small" /></ListItemIcon>
+              <ListItemText primary={`${deleted} ${translatePlural('datasource.tooltip.deleted', deleted)}`} />
+            </ListItem>
+          )}
+        </List>
+      </List>
+    </div>
+  );
+}
```
Comment (mabhub) [inline src/modules/RA/DataSource/components/StatusChip/Tooltip.js:117]:
```suggestion
};

export default TooltipStatus;
```

File: src/modules/RA/DataSource/components/StatusChip/Tooltip.js
Comment (mabhub) [inline src/modules/RA/DataSource/components/StatusChip/Tooltip.js:1]: Wouldn't it more logical giving the file the same name as the component?

File: src/modules/RA/DataSource/components/ReportTab.js [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+    borderRadius: '5px',
+    backgroundColor: '#f5f6fa',
+  },
+  nested: {
+    paddingLeft: theme.spacing(8),
+  },
+}));
+
+
+export default function ReportTab ({ report, translate }) {
```
Comment (mabhub) [inline src/modules/RA/DataSource/components/ReportTab.js:46]:
```suggestion
const ReportTab = ({ report, translate }) => {
```

File: src/modules/RA/DataSource/components/ReportTab.js [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+                  {renderRow}
+                </FixedSizeList>
+              </List>
+            </>
+          )}
+        </div>
+      </List>
+    </div>
+  );
+}
```
Comment (mabhub) [inline src/modules/RA/DataSource/components/ReportTab.js:191]:
```suggestion
};

export default ReportTab;
```

File: src/modules/RA/DataSource/components/ReportTab.js [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+import FlashOnOutlinedIcon from '@material-ui/icons/FlashOnOutlined';
+import AddCircleOutlineOutlinedIcon from '@material-ui/icons/AddCircleOutlineOutlined';
+import DeleteOutlinedIcon from '@material-ui/icons/DeleteOutlined';
+import AssessmentOutlinedIcon from '@material-ui/icons/AssessmentOutlined';
+import CreateOutlinedIcon from '@material-ui/icons/CreateOutlined';
+import WarningOutlinedIcon from '@material-ui/icons/WarningOutlined';
+import PlayArrowOutlinedIcon from '@material-ui/icons/PlayArrowOutlined';
+import DoneAllOutlinedIcon from '@material-ui/icons/DoneAllOutlined';
+import FeedbackIcon from '@material-ui/icons/Feedback';
+import DescriptionIcon from '@material-ui/icons/Description';
```
Comment (mabhub) [inline src/modules/RA/DataSource/components/ReportTab.js:14]:
```suggestion
import {
  FlashOnOutlined as FlashOnOutlinedIcon,
  AddCircleOutlineOutlined as AddCircleOutlineOutlinedIcon,
  DeleteOutlined as DeleteOutlinedIcon,
  AssessmentOutlined as AssessmentOutlinedIcon,
  CreateOutlined as CreateOutlinedIcon,
  WarningOutlined as WarningOutlinedIcon,
  PlayArrowOutlined as PlayArrowOutlinedIcon,
  DoneAllOutlined as DoneAllOutlinedIcon,
  Feedback as FeedbackIcon,
  Description as DescriptionIcon,
} from '@material-ui/icons';
```

**Review by mabhub — CHANGES_REQUESTED** [review]

File: src/modules/RA/DataSource/components/StatusChip/Tooltip.js
Comment (CFarcy) [reply src/modules/RA/DataSource/components/StatusChip/Tooltip.js:1]: yes, absolutely

Comment (CFarcy) [reply src/modules/RA/DataSource/components/StatusChip/Tooltip.js:1]: done !

**Review by mabhub — APPROVED** [review]

**Pull request merged**